### ISSUE TITLE
Jasper can't understand a word I'm saying... even the command "jasper" he only gets about 1 in 20 times

### ISSUE THREAD

**Issue by bryanbecker** [jasperproject/jasper-client#362]
Everything is set up correctly, I think.

I've also separately tested my mic by `arecord -vv -fdat foo.wav` and then `aplay -vv foo.wav` and it seems quite clear to me, although with a small layer of "staticy" background buzz

I'm using pocketsphinx, with the default hmm and fst_model (fst from phonetisaurus download, and the hmm that comes with pocketsphinx)

he constantly just hears "say", "with", and "now": 

here is a sample:
```
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['SAY']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['WITH']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['JASPER']
INFO:client.stt:Transcribed: ['NO WEATHER TIME']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['NOW']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['SAY']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['IN']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['NOW']
INFO:client.stt:Transcribed: ['']
INFO:client.stt:Transcribed: ['NOW']
INFO:client.stt:Transcribed: ['']
```

the "no weather time," was me asking "what time is it?"

Is the problem my voice?  Do I need to "retrain" a model with my own voice?

**Comment by glop102** [comment]
Depending on the amount of static, it could be overpowering your voice to the system.
Are you using the dev branch of jasper? The master brance cuts the mic every few seconds which screws with timing when you say things. Also, make sure you are close enough to your mic.

I personally am using the google STT for the moment trying to get things to work properly for me. Try using that first to remove the STT as the possible weak link.

**Comment by RyanDoc** [comment]
Hey guys! I did quick start and the default is "Sphinx." I also noticed a very poor voice recognition. I figured I would try to setup a different STT, even the [Documentation for Jasper] (http://jasperproject.github.io/documentation/configuration/#pocketsphinx-stt) said "Sphinx" was poor. I believe this will be the solution, but now I am having trouble with the syntax of the profile.yml. I tried using wit.ia and I believe it will work better, the error I get is:

*ScannerError: while scanning for next token*
*found character '\t' that cannot start token*
*(indent) in "/home/pi/.jasper/profile.yml", line 11, column 1*

So I know this is saying I have some sort of unnecessary TAB/indent in line 11 of my profile.yml. Here is what my Profile.yml looks like:

line 9:  *stt_engine: wit.ai*
line 10: *witai-stt:*
line 11: *(indent) access_token: (MY_ACCESS_TOKEN)* 

I am positive my access token is correct. I believe my problem is just syntax in the .yml file. I tried formatting it just like the Jasper Documentation said, but I think I made a mistake. Any suggestions?
